### PR TITLE
[SOAR-9449] AWS EC2 - Bump Python to 3 and update to insightconnect r…

### PR DESCRIPTION
**Pull request by jkaluza-rapid7**
### Description

  - Update python version
  - update insightconnect_runtime version to 4

#### Integration Tests

Tested manually in postman as a standalone server

#### Unit Tests

No unit-tests. We will have a separate PR for them.

### CODE REVIEW

File: plugins/aws_ec2/help.md [hunk 1]
```diff
@@ -14,17 +14,31 @@ to general management of EC2 instances.
 * An AWS account with admin permissions for EC2
 * An AWS Access Key ID and Secret Access Key
 
+# Supported Product Versions
+
+_There are no supported product versions listed._
+
```
Comment (igorski-r7) [inline plugins/aws_ec2/help.md:20]: Kindly please `make regenerate` to fit `supported_versions` field form plugin.spec.yaml file 😃

File: plugins/aws_ec2/help.md [hunk 2]
```diff
@@ -173,43 +286,69 @@ Removes one or more ingress rules from a security group. See [http://docs.aws.am
...
+```
+{
+  "dry_run": true,
+  "group_id": "sg-123456",
+  "group_name": "groupname",
+  "ip_permissions": [
+    "{ \"from_port\": 8080, \"ip_protocol\": \"tcp\", \"ip_ranges\" : [{ \"cidr_ip\": \"10.20.0.0/27\", \"description\": \"Blah\" }], \"to_port\": 8090}"
+  ]
+}
```
Comment (igorski-r7) [inline plugins/aws_ec2/help.md:306]: Same comment for all `Example inputs` 😃 . Wouldn't it look much better this way?
```suggestion
{
  "dry_run": true,
  "group_id": "sg-123456",
  "group_name": "groupname",
  "ip_permissions": [
    {
      "from_port": 8080,
      "ip_protocol": "tcp",
      "ip_ranges": [
        {
          "cidr_ip": "10.20.0.0/27",
          "description": "Blah"
        }
      ],
      "to_port": 8090
    }
  ]
}
```

**Review by igorski-r7 — CHANGES_REQUESTED** [review]

**Review by cmcnally-r7 — APPROVED** [review]

**Review by igorski-r7 — APPROVED** [review]

File: plugins/aws_ec2/icon_aws_ec2/actions/authorize_security_group_egress/action.py [hunk 3]
```diff
@@ -1,16 +1,15 @@
-import komand
 from .schema import AuthorizeSecurityGroupEgressInput, AuthorizeSecurityGroupEgressOutput
```
Comment (mberezin-r7) [inline plugins/aws_ec2/icon_aws_ec2/actions/authorize_security_group_egress/action.py:1]: For each action, please add

```suggestion
from .schema import AuthorizeSecurityGroupEgressInput, AuthorizeSecurityGroupEgressOutput, Component
```
...

File: plugins/aws_ec2/icon_aws_ec2/actions/authorize_security_group_egress/action.py [hunk 4]
```diff
@@ -1,16 +1,15 @@
...
 # Custom imports below
-from komand_aws_ec2.util.common import AWSAction
+from icon_aws_ec2.util.common import AWSAction
 
 
 class AuthorizeSecurityGroupEgress(AWSAction):
     def __init__(self):
         super().__init__(
             name="authorize_security_group_egress",
             description="[EC2-VPC only] Adds one or more egress rules to a security group for use with a VPC",
```
Comment (mberezin-r7) [inline plugins/aws_ec2/icon_aws_ec2/actions/authorize_security_group_egress/action.py:11]: ...
and please add
```suggestion
            description=Component.DESCRIPTION
```

**Review by mberezin-r7 — CHANGES_REQUESTED** [review]

File: plugins/aws_ec2/icon_aws_ec2/actions/authorize_security_group_egress/action.py (hunk 3 above)
Comment (jkaluza-rapid7) [reply plugins/aws_ec2/icon_aws_ec2/actions/authorize_security_group_egress/action.py:1]: Thanks, good catch 👍

**Pull request closed without merging**